src/main/resources/db/MTB dml.sql

```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
 insert into gender (code)
 values ('MALE'),
        ('FEMALE');
+
+insert into size (code)
```

Comment:
size table does not have a code. Change to value here

Reply:
will wait for changes